ios/Classes/SwiftDevCycleFlutterClientSdkPlugin.swift

```diff
@@ -4,278 +4,355 @@ import UIKit
-          } else {
-            callbackArgs["variables"] = self.variablesToMap(variables: variables ?? [:])
-          }
-          self.channel.invokeMethod("userIdentified", arguments: callbackArgs)
-        })
-      }
-      result(nil)
-    case "resetUser":
-      try? self.dvcClient?.resetUser(callback: { error, variables in
-        callbackArgs["callbackId"] = callbackId
-        if (error != nil) {
-          callbackArgs["error"] = "\(String(describing: error))"
-        } else {
-          callbackArgs["variables"] = self.variablesToMap(variables: variables ?? [:])
-        }
-        self.channel.invokeMethod("userReset", arguments: callbackArgs)
-      })
-      result(nil)
-    case "allFeatures":
-        result(featuresToMap(features: self.dvcClient?.allFeatures() ?? [:]))
-    case "allVariables":
-        result(variablesToMap(variables: self.dvcClient?.allVariables() ?? [:]))
-    case "getPlatformVersion":
-      result("iOS " + UIDevice.current.systemVersion)
-    case "variable":
-      if let dvcClient = self.dvcClient, let varKey = args?["key"] as? String, let varDefaultValue = args?["defaultValue"] as? Any {
-        let variable = dvcClient.variable(key: varKey, defaultValue: varDefaultValue)
-        if !self.variableUpdates.contains(where: { $0.key == variable.key }) {
-          variable.onUpdate(handler: { newValue in
-            callbackArgs["key"] = variable.key
-            callbackArgs["value"] = variable.value
-            self.channel.invokeMethod("variableUpdated", arguments: callbackArgs)
-          })
-          self.variableUpdates[variable.key] = variable
-        }
-        let codecVariable = dvcVariableToMap(variable: variable)
-        result(codecVariable)
-      } else {
-        result(nil)
-      }
-    case "track":
-      if let event = args?["event"] as? [String: Any], let dvcEvent = getEventFromDict(dict: event) {
-        self.dvcClient?.track(dvcEvent)
-      }
-    case "flushEvents":
-      self.dvcClient?.flushEvents(callback: { error in
-        callbackArgs["callbackId"] = callbackId
-        if (error != nil) {
-          callbackArgs["error"] = "\(String(describing: error))"
-        }
-        self.channel.invokeMethod("eventsFlushed", arguments: callbackArgs)
-      })
-      result(nil)
-    default:
-      result(FlutterMethodNotImplemented)
-    }
-  }
-  
-  private func getUserFromDict(dict: [String: Any?]) -> DVCUser? {
-    let userBuilder = DVCUser.builder()
-    
-    if let userId = dict["userId"] as? String {
-      userBuilder.userId(userId)
-    }
-
-    if let isAnonymous = dict["isAnonymous"] as? Bool {
-      userBuilder.isAnonymous(isAnonymous)
-    }
-    
-    if let email = dict["email"] as? String {
-      userBuilder.email(email)
-    }
-    
-    if let name = dict["name"] as? String {
-      userBuilder.name(name)
+    private init(channel: FlutterMethodChannel) {
+        self.channel = channel
```

Comment:
a lot of this is just indenting, will comment the sections that are new / changed